Clarify upload location for Builder Image provisioning (#3958)
theforeman/foreman-documentation: guides/common/modules/proc_provisioning-from-a-builder-image-on-an-http-server.adoc

:_mod-docs-content-type: PROCEDURE

[id="provisioning-from-a-builder-image-on-an-http-server"]
= Provisioning from a {LoraxCompose} image on an HTTP server

You can upload a {LoraxCompose} image to an HTTP server and use this image to provision hosts by using network boot.

ifndef::satellite[]
[NOTE]
====
This feature is available for {EL} hosts only.
====
endif::[]

When you provision from a {LoraxCompose} image, Anaconda installer partitions disks, downloads and mounts the image, and copies files over to a host.

.Prerequisites
* An existing TAR image created by using {LoraxCompose} in {the-Cockpit}.
Ensure that your blueprint to build the image includes the xref:packages-required-in-a-builder-image[required packages].

.Procedure

> . Copy the image to an existing HTTP server which installed hosts can reach.

Author: ```suggestion
. Copy the image to an existing HTTP server that your hosts can reach.
```

[id="provisioning-from-a-builder-image-on-an-http-server"]
= Provisioning from a {LoraxCompose} image on an HTTP server

You can upload a {LoraxCompose} image to an HTTP server and use this image to provision hosts.

ifndef::satellite[]
[NOTE]
====
This feature is available for {EL} hosts only.
====
endif::[]

When you configure {Project} to provision from a {LoraxCompose} image, Anaconda installer partitions disks, downloads and mounts the image, and copies files over to a host.

.Prerequisites
* An existing TAR image created by using {LoraxCompose}.
Ensure that your blueprint to build the TAR image includes a kernel package.

.Procedure
. Copy the TAR image to an existing HTTP server which installed hosts can reach.
. In the {ProjectWebUI}, navigate to *Configure* > *Host Groups*, and select the host group that you want to use.
. Click the *Parameters* tab, and then click *Add Parameter*.
. In the *Name* field, enter `kickstart_liveimg`.
. From the *Type* list, select *string*.

> . In the *Value* field, enter the URL of the image location, such as `http://_example.com_/_path_/_image_name_`.

Author: I checked the Kickstart docs and the [liveimg command](https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#liveimg) does support HTTPS in the URL. Let's do it.
